src/android/platform.cpp

```diff
@@ -93,4 +93,27 @@ namespace realm {
                           s_default_realm_directory + "/*.realm.lock";
         system(cmd.c_str());
     }
+
+    static void remove_directory(const std::string &path)
+    {
+        std::string cmd_clear_dir = "rm " + path + "/*";
+        system(cmd_clear_dir.c_str());
```

Comment:
We should change this, but if it's what we already did then let's go with it for now.

Reply:
Yeah, we're already using 'rm' on Android.